run_project_tests.py

```diff
@@ -1123,6 +1123,7 @@ def __init__(self, category: str, subdir: str, skip: bool = False, stdout_mandat
         TestCategory('platform-osx', 'osx', not mesonlib.is_osx()),
         TestCategory('platform-windows', 'windows', not mesonlib.is_windows() and not mesonlib.is_cygwin()),
         TestCategory('platform-linux', 'linuxlike', mesonlib.is_osx() or mesonlib.is_windows()),
+        TestCategory('platform-android', 'android', not mesonlib.is_android()),
```

Comment:
This will never actually run. `is_android` means "is the machine I'm currently running on an Android machine", which would only be true when running on an Android device and most (all?) builds for Android are done by cross compilation. I don't think we have the Android NDK installed on any CI machines either so the test could not be run in any case.

Reply:
Maybe just add a comment like:

    # FIXME, does not actually run in CI, change to run the test if an Android cross toolchain is detected.